elastic/kibana
# [Security Solution] Invalidate prebuilt rules status after package upgrade or installation

**Resolves: https://github.com/elastic/kibana/issues/150306**

## Summary

Fixes the Load Prebuilt rules button not visible when users visit the rules management page for the first time (no prebuilt detection rules package installed).

## Steps to test

1. Ensure that the detection engine package is not installed: <img src="https://user-images.githubusercontent.com/1938181/216949857-a49731f3-535e-4813-95d3-62b2995500bb.png" width="600" />
2. Navigate to the rules management page.

### Previously

The "Load Elastic Prebuilt Rules" button is not visible, and users cannot install prebuilt rules.

<img src="https://user-images.githubusercontent.com/1938181/216950430-4c57a3ad-3146-40a0-82c2-08c4fc2f65c3.png" width="600" />

### With the fix

Users now see loading animation, indicating that the package installation happens in the background. Once the package installation finishes, users see the Load Prebuilt rules button appear.

https://user-images.githubusercontent.com/1938181/217585144-879fe288-0ede-4e01-b585-6aced1d89379.mov
